File: codes/ruby/object_entries.rb
require "json"

print("\n", "# JavaScript-like Object.entries() in Ruby Hash")

friend = {
    "name" => "Alisa",
    "country" => "Finland",
    "age" => 25
}
print("\n", "friend: ", JSON.pretty_generate(friend, { "indent": " " * 4 }))

print("\n", "friend entries: ", JSON.generate(friend.entries()).gsub(",", ", "))
# friend entries: [["name", "Alisa"], ["country", "Finland"], ["age", 25]]

print("\n", "friend entries: ", JSON.generate(friend.entries).gsub(",", ", "))
# friend entries: [["name", "Alisa"], ["country", "Finland"], ["age", 25]]

File: codes/python/object_entries.py
import json

print('\n# JavaScript-like Object.entries() in Python dictionary')

friend = {
    "name": "Alisa",
    "country": "Finland",
    "age": 25
}
print(f'friend: {json.dumps(friend, indent=4)}')

print(f'friend entries: {json.dumps(list(friend.items()))}')
# friend entries: [["name", "Alisa"], ["country", "Finland"], ["age", 25]]
